ee/clickhouse/queries/funnels/test/breakdown_cases.py

```diff
@@ -211,39 +191,40 @@ def test_funnel_step_breakdown_event_with_other(self):
-                [
-                    {"event": "sign up", "day": 1, "hour": 12},
-                    {"event": "play movie", "day": 1, "hour": 13},
-                    {"event": "buy", "day": 1, "hour": 15},
+            events_by_person = {
+                "person1": [
+                    {"event": "sign up", "timestamp": datetime(2020, 1, 1, 12), "properties": {"$browser": "Chrome"}},
+                    {
+                        "event": "play movie",
+                        "timestamp": datetime(2020, 1, 1, 13),
+                        "properties": {"$browser": "Chrome"},
+                    },
+                    {"event": "buy", "timestamp": datetime(2020, 1, 1, 15), "properties": {"$browser": "Chrome"}},
                 ],
-                {"key": "val", "$browser": "Chrome"},
-            )
-
-            person2 = _create_person(distinct_ids=["person2"], team_id=self.team.pk)
-            self.a_journey_for(
-                "person2",
-                [{"event": "sign up", "day": 2, "hour": 14}, {"event": "play movie", "day": 2, "hour": 16},],
-                {"key": "val", "$browser": "Safari"},
-            )
-
-            person3 = _create_person(distinct_ids=["person3"], team_id=self.team.pk)
-            self.a_journey_for(
-                "person3", [{"event": "sign up", "day": 2, "hour": 14},], {"key": "val", "$browser": "Safari"}
-            )
-
-            person4 = _create_person(distinct_ids=["person4"], team_id=self.team.pk)
-            self.a_journey_for(
-                "person4", [{"event": "sign up", "day": 2, "hour": 14},], {"key": "val", "$browser": "random"}
-            )
+                "person2": [
+                    {"event": "sign up", "timestamp": datetime(2020, 1, 2, 14), "properties": {"$browser": "Safari"}},
+                    {
+                        "event": "play movie",
+                        "timestamp": datetime(2020, 1, 2, 16),
+                        "properties": {"$browser": "Safari"},
+                    },
+                ],
+                "person3": [
+                    {"event": "sign up", "timestamp": datetime(2020, 1, 2, 14), "properties": {"$browser": "Safari"}},
+                ],
+                "person4": [
+                    {"event": "sign up", "timestamp": datetime(2020, 1, 2, 14), "properties": {"$browser": "random"}},
+                ],
+                "person5": [
+                    {
+                        "event": "sign up",
+                        "timestamp": datetime(2020, 1, 2, 15),
+                        "properties": {"$browser": "another one"},
+                    },
+                ],
+            }
 
-            person5 = _create_person(distinct_ids=["person5"], team_id=self.team.pk)
-            self.a_journey_for(
-                "person5", [{"event": "sign up", "day": 2, "hour": 15},], {"key": "val", "$browser": "another one"}
-            )
+            people = journeys_for(events_by_person, self.team)
```

Comment:
I missed the last PR as it got merged really quickly but this is a great opportunity to start batch creating and enhancing tests. refer here: https://github.com/PostHog/posthog/pull/6725

Reply:
Yes, we should be able to perform some perf improvements. It would be good to try and keep the same invariants that are provided by `from ee.clickhouse.models.event import create_event` or however we're otherwise creating events normally.